parts/linux/cloud-init/artifacts/init-aks-custom-cloud-mariner.sh

```diff
@@ -16,7 +16,12 @@ cp /root/AzureCACertificates/*.crt /etc/pki/ca-trust/source/anchors/
 
 cloud-init status --wait
 
-# TODO - Set the repoDepotEndpoint in a .repo file if package update becomes necessary
+marinerRepoDepotEndpoint="$(echo "${REPO_DEPOT_ENDPOINT}" | sed 's/\/ubuntu//')"
```

Comment:
could we add an error for if this is empty in custom cloud case or something? might be a good sanity check

Reply:
@hbeberman from your side, what would be the expected behavior here if that happens? Wrap this around an if-block and don't update repoDepot endpoints? Or throw some sort of error message?

Reply:
I think throwing an error would be preferrable, you don't want to leave the VMs in a state where they assume they'll be getting automatic package updates, but the repos aren't actually pointing to the correct mirrors.